phpcs-server/src/linter.ts

```diff
@@ -152,13 +152,18 @@ export class PhpcsPathResolver {
 
 		let pathSeparator = /^win/.test(process.platform) ? ";" : ":";
 		let globalPaths = process.env.PATH.split(pathSeparator);
-		globalPaths.forEach(globalPath => {
+		globalPaths.every(globalPath => {
```

Comment:
@ADmad I think it would be more accurate with globalPaths.some returning true if a match is found otherwise false.

Reply:
I am not a js dev. Feel free to optimise the code as you see fit. All I can say is this patch fixes the problem of extension not using phpcs found in global path.